src/main/java/org/prebid/server/analytics/reporter/liveintent/LiveIntentAnalyticsReporter.java

```diff
@@ -0,0 +1,195 @@
+        final Optional<ExtRequestPrebid> requestPrebid = Optional.ofNullable(bidRequest.getExt())
+                .flatMap(ext -> Optional.of(ext.getPrebid()));
+
+        final Optional<Activity> activity = getActivities(auctionContext);
+        final boolean isEnriched = isEnriched(activity);
+        final Float treatmentRate = getTreatmentRate(activity);
+        final Long timestamp = requestPrebid.map(ExtRequestPrebid::getAuctiontimestamp).orElse(0L);
+
+        final List<PbsjBid> pbsjBids = CollectionUtils.emptyIfNull(bidResponse.getSeatbid()).stream()
+                .map(SeatBid::getBid)
+                .flatMap(Collection::stream)
+                .map(bid -> buildPbsjBid(bidRequest, bidResponse, bid, isEnriched, treatmentRate, timestamp))
+                .filter(Optional::isPresent)
+                .map(Optional::get)
+                .toList();
+
+        try {
+            return httpClient.post(
+                    new URIBuilder(URI.create(properties.getAnalyticsEndpoint()))
+                            .setPath("/analytic-events/pbsj-bids")
+                            .build()
+                            .toString(),
+                    jacksonMapper.encodeToString(pbsjBids),
+                    properties.getTimeoutMs()
+                 ).mapEmpty();
+        } catch (Exception e) {
+            logger.error("Error processing event: {}", e.getMessage());
+            return Future.failedFuture(e);
+        }
+    }
+
+    private Optional<Activity> getActivities(AuctionContext auctionContext) {
+        return Optional.ofNullable(auctionContext)
+                .map(AuctionContext::getHookExecutionContext)
+                .map(HookExecutionContext::getStageOutcomes)
+                .map(stages -> stages.get(Stage.processed_auction_request))
+                .stream()
+                .flatMap(Collection::stream)
+                .filter(stageExecutionOutcome -> "auction-request".equals(stageExecutionOutcome.getEntity()))
+                .map(StageExecutionOutcome::getGroups)
+                .flatMap(Collection::stream)
+                .map(GroupExecutionOutcome::getHooks)
+                .flatMap(Collection::stream)
+                .filter(hook ->
+                        LIVEINTENT_HOOK_ID.equals(hook.getHookId().getModuleCode())
+                                && hook.getStatus() == ExecutionStatus.success
+                )
+                .map(HookExecutionOutcome::getAnalyticsTags)
+                .filter(Objects::nonNull)
+                .map(Tags::activities)
+                .filter(Objects::nonNull)
+                .flatMap(Collection::stream)
+                .filter(Objects::nonNull)
+                .findFirst();
+    }
+
+    private Float getTreatmentRate(Optional<Activity> activity) {
+        return activity.stream()
+                .flatMap(a -> a.results().stream())
+                .findFirst()
+                .map(a -> a.values().at("treatmentRate").floatValue())
```

Comment:
First check if this property is present

Reply:
Done in https://github.com/prebid/prebid-server-java/pull/4154/commits/58fc8c0e317de0aa062fddda86b40ee1758acca6